.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: scala
+
+scala:
+
+   - 2.10.0
+
+script: sbt test
```

Comment:
I'm working on it. Also trying to repair the commit history so it doesn't take 4 commits to add this file...